src/App.js

```diff
@@ -27,42 +29,14 @@ class App extends React.Component {
         <div className="container">
           <p className="text-center mb-4">Browse all the recipes here, select a category or search for an ingredient.</p>
           <div className="row">
-            <div className="col-sm-6 col-lg-4 mb-4">
-              <div className="card">
-                <img src="..." className="card-img-top" alt="..." />
-                <div className="card-body">
-                  <h5 className="card-title">Card title</h5>
-                  <p className="card-text">Some quick example text to build on the card title and make up the bulk of the card’s content.</p>
-                </div>
-              </div>
-            </div>
-            <div className="col-sm-6 col-lg-4 mb-4">
-              <div className="card">
-                <img src="..." className="card-img-top" alt="..." />
-                <div className="card-body">
-                  <h5 className="card-title">Card title</h5>
-                  <p className="card-text">Some quick example text to build on the card title and make up the bulk of the card’s content.</p>
-                </div>
-              </div>
-            </div>
-            <div className="col-sm-6 col-lg-4 mb-4">
-              <div className="card">
-                <img src="..." className="card-img-top" alt="..." />
-                <div className="card-body">
-                  <h5 className="card-title">Card title</h5>
-                  <p className="card-text">Some quick example text to build on the card title and make up the bulk of the card’s content.</p>
-                </div>
-              </div>
-            </div>
-            <div className="col-sm-6 col-lg-4 mb-4">
-              <div className="card">
-                <img src="..." className="card-img-top" alt="..." />
-                <div className="card-body">
-                  <h5 className="card-title">Card title</h5>
-                  <p className="card-text">Some quick example text to build on the card title and make up the bulk of the card’s content.</p>
-                </div>
-              </div>
-            </div>
+            {recipes.map((recipe, i) =>
```

Comment:
You receive the recipe here, there is no need to do another lookup, you can use it straight away from here.
eg. `imageUrl={recipe.image}`

Reply:
can't believe I actually did it this way 🤦